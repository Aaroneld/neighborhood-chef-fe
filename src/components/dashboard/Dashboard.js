import React from "react";
import Sidebar from "./Sidebar";

const Dashboard = () => {
  return (
    <div>
      <h1>Dashboard Component</h1>
      <Sidebar />
    </div>
  );
};

export default Dashboard;
