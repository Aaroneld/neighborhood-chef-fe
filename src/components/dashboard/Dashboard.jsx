import React from 'react';
// import Feed from "./Feed";
import RecentEvents from './recent-events/RecentEvents';

const Dashboard = () => {
    return (
        <div style={{ display: 'flex', width: '100%' }}>
            <div
                style={{
                    display: 'flex',
                    flexDirection: 'column',
                    width: '100%',
                    alignItems: 'center',
                }}
            >
                <div>
                    <RecentEvents />
                </div>
            </div>
        </div>
    );
};

export default Dashboard;
