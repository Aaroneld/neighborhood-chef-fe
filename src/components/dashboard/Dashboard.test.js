import React from "react";
import Dashboard from "./Dashboard.js";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import { BrowserRouter } from "react-router-dom";

describe("Test dashboard static properties", () => {
  beforeEach(() => {
    const DashboardComponent = render(
      <BrowserRouter>
        <Dashboard />
      </BrowserRouter>
    );
  });

  test("Dashboard component renders", () => {
    const secondDiv = document.querySelector(".component-main");
    expect(secondDiv.toBeInDocument);
  });
});
