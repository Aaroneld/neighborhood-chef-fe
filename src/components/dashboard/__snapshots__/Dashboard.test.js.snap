// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard component renders 1`] = `
<h1>
  Dashboard Component
</h1>
`;
